### ISSUE TITLE
vi syntax highlighting

### ISSUE THREAD

**Issue by wangjiawen2013** [xonsh/xonsh#3942]
Hi,
It seems that my vi editor doesn't support syntax highlighting in .xsh (xonsh script), while .sh scripts is suppoted. who can tell how to highlight it in vi editor ?

**Comment by anki-code** [comment]
Hi @wangjiawen2013 ! You can find supported editors in [Editors and IDE](https://xon.sh/editors.html) page.

There is [third party solution for vim](https://xon.sh/editors.html#vim).

**Comment by dyuri** [comment]
Or - since `.xsh` files are basically python - you can just put something like this into your `.vimrc`:
```vim
augroup xsh
  au! BufNewFile,BufRead *.xsh   set filetype=python
augroup END
```

**Comment by anki-code** [comment]
Closed with reference to #2490 and the solutions above.
